assets/js/react/GameStateEditor/reducers/entityForm.js

```diff
@@ -7,27 +8,41 @@ import {
   ADD_ENTITY_SUCCESS,
   ADD_ENTITY_FAILURE,
 } from "../actions/entityForm"
+import { EDIT_ENTITY, CLOSE_EDIT_PANE } from "../../actions/app"
+import { convertBehaviorsToHash } from "../selectors/entityForm"
 
 export const defaultState = {
   availableBehaviors: [],
   entity: {},
 }
 
 export default function(state = defaultState, action) {
+  const data = camelizeKeys(action.data)
   switch (action.type) {
+    case CLOSE_EDIT_PANE:
+      return mergeDeepRight(state, { entity: {} })
+
+    case EDIT_ENTITY:
+      if (data) {
+      const entity = convertBehaviorsToHash(data.body.data.entity)
+        return mergeDeepRight(state, { entity, editStarted: (new Date()) })
```

Comment:
by adding `editStarted` we can send this to the `react` component as a`key` making it easy to repopulate the form every time we type "/edit goose".  in the game.